backend/telemetry/Cargo.toml

```diff
@@ -18,6 +18,10 @@ tracing-opentelemetry = { version = "0.28.0" }
 tracing-subscriber = { version = "0.3.19" }
 url = { workspace = true }
 
+[dev-dependencies]
+tokio = { workspace = true }
+opentelemetry-proto = { version = "0.28.0" }
+tonic = { version = "0.12.3" }
```

Comment:
Another nitpick, take it or leave it: seems like it's general convention in `Cargo.toml` files to order the crates alphabetically